### PR TITLE
Middle section and about me section designed like Figma template

### PR DESCRIPTION
**Pull request by lexoxo**
This branch brings some styles to design the portfolio like the third template of Figma templates

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,8 +26,148 @@ <h1>Hey there. I'm Axel<br><span id="second-h1">I'm a software developer</span><
...
+
+<section class="recents-work">
+    <h2 class="recent-work">My Recent Works</h2>
+    <div class="under-bar"></div>
+
+    <div class="works">
+        <div class="work">
+            <img src="images/default.png" alt="work logo">
+           <div class="work-text">
+            <p class="work-description">Multi-Post Stories Gain+Glory</p>
```
Comment (ReshyResh) [inline index.html:40]: - [ ] Please make sure the title of each work card title uses an `hx` tag, for example `h2`.

File: index.html [hunk 2]
```diff
@@ -26,8 +26,148 @@ <h1>Hey there. I'm Axel<br><span id="second-h1">I'm a software developer</span><
...
+                                </div>
+                                
+                            </div>
+                    </div>
+                    
+
+</section>
+
+<section class="about-myself">
+    <h1 class="about-me">About me</h1>
```
Comment (ReshyResh) [inline index.html:131]: - [ ] For both semantic and SEO purposes, you should always avoid having multiple `h1` tags in a single page, please make sure to change this to another tag and style it accordingly.

File: index.html [hunk 3]
```diff
@@ -26,8 +26,148 @@ <h1>Hey there. I'm Axel<br><span id="second-h1">I'm a software developer</span><
...
+    <p class="about-description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+    </p>
+<div class="resume-button">Get My Resume</div>
```
Comment (ReshyResh) [inline index.html:135]: - [ ] I suggest you change this to a `button` tag, as this is its function. Good job on implementing the interactions!

File: index.html [hunk 4]
```diff
@@ -26,8 +26,148 @@ <h1>Hey there. I'm Axel<br><span id="second-h1">I'm a software developer</span><
...
         <li><i class="fa fa-hand-peace-o"></i></li>
         </ul>
-       <img class="more-img" src="images/more.png" alt=""/>
-      
+       <img class="more-img" src="images/more2.svg" alt=""/>  
+</section>
+
+<section class="recents-work">
+    <h2 class="recent-work">My Recent Works</h2>
+    <div class="under-bar"></div>
```
Comment (ReshyResh) [inline index.html:34]: - [ ] Please make sure to add a bit more spacing to those elements as they're currently overlapping each other.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @lexoxo , 

### Good job so far, your portfolio looks great!
  - ✔ Correct use of Flexbox and Grid to lay out elements.
  - ✔ Descriptive commit messages and professional looking repository.
  - ✔ No linter errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Please check the comments under the review._


Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (lexoxo) [reply index.html:34]: Thanks for the feedback

**Review by ReshyResh — APPROVED** [review]
# Hi again @lexoxo , 

### Good job addressing the changes requested previously!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: index.html (hunk 1 above)
Comment (EzekielUtshudi) [reply index.html]: Your work is well structured but lease next time have more descriptive comments to help others understand your codes.

**Pull request merged**